src/main/scala/Collection.scala

```diff
@@ -0,0 +1,36 @@
+class Collection {
+  def ExSet(){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter